plugins/meta/tuning/tuning.go

```diff
@@ -90,6 +163,26 @@ func cmdAdd(args *skel.CmdArgs) error {
 				return err
 			}
 		}
+
+		var err error
```

Comment:
I can't see what this gives, over `err :=` on the three subsequent uses

Reply:
Removed.